### PR TITLE
CLOUDP-260774: Add some example commands

### PR DESCRIPTION
**Pull request by stefan4h**
- add some example commands

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -7,6 +7,4 @@ github:
 binary: example
 commands: 
     testplugin:
```
Comment (tibulca) [inline manifest.yml:9]:
```suggestion
    example: 
```

File: manifest.yml [hunk 2]
```diff
@@ -7,6 +7,4 @@ github:
 binary: example
 commands: 
     testplugin: 
-        description: this is the first test command
-    testplugin2:
-        description: this is the second test command
\ No newline at end of file
+        example: Root command of the atlas cli plugin example
```
Comment (tibulca) [inline manifest.yml:10]:
```suggestion
        description: Root command of the atlas cli plugin example
```

File: manifest.yml (hunk 1 above)
Comment (tibulca) [reply manifest.yml:9]: I think this is the name of the command, right?

Comment (stefan4h) [reply manifest.yml:9]: yes I changed the wrong value by accident. Fixed it now

**Review by tibulca — APPROVED** [review]

**Pull request merged**